tests/components/matter/test_init.py

```diff
@@ -412,13 +435,35 @@ async def test_update_addon(
 
 
 # This tests needs to be adjusted to remove lingering tasks
-@pytest.mark.parametrize("expected_lingering_tasks", [True])
+@pytest.mark.parametrize(
+    (
+        "expected_lingering_tasks",
```

Comment:
Keep this parameter in a separate decorator since its value is constant and we want to be able to clean it up easily in the future.